### PR TITLE
Feature scoring system

### PR DESCRIPTION
**Pull request by marcelkuipers**
This is a rough implementation of how the score should be represented visually and how points can be added. Please note that the point of this PR is not yet to implement a routine for adding points, only the functionality.

### CODE REVIEW

File: ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestEnvironment.java [hunk 1]
```diff
@@ -29,44 +30,49 @@
...
+
 	protected Node root;
 	protected AssetManager assetManager;
-	
+	private HUDController hudController;
+
+
 	/**
 	 * Initialises the environment field for testing.
+	 * @throws Exception
```
Comment (NilsHullegien) [inline ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestEnvironment.java]: Missing javadoc description for the Exception.

File: ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestMainEnvironment.java [hunk 2]
```diff
@@ -49,14 +50,16 @@
     private Spatial model;
     private RenderManager renderManager;
     private Camera cam;
+    private HUDController hudController;
 
     /**
      * This method starts every time a new test case starts.
      */
```
Comment (NilsHullegien) [inline ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestMainEnvironment.java:58]: Missing @Exception tag in javadoc

**Comment by NilsHullegien** [comment]
Besides my very minor comments, I approve of this code.

**Comment by marcelkuipers** [comment]
Fixed that now.

**Comment by NilsHullegien** [comment]
Looks good now!

File: ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestEnvironment.java [hunk 3]
```diff
@@ -29,44 +30,49 @@
...
 	 * Initialises the environment field for testing.
+	 * @throws Exception exception that is thrown.
 	 */
 	@Before
-	public void setUp() {
+	public void setUp() throws Exception {
 		this.environment = new Environment();
-		
+
+        hudController = Mockito.mock(HUDController.class);
```
Comment (ThatFlint) [inline ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestEnvironment.java]: The indenting doesn't match here. There's spaces in front of line 51 and 55, while the other lines have tabs.

**Comment by marcelkuipers** [comment]
I fixed the indenting.

**Comment by ThatFlint** [comment]
's gud

**Pull request merged**